DOL.WHD.Section14c.Log/Controllers/APIActivityLogsController.cs

```diff
@@ -44,22 +36,16 @@ public IQueryable<APIActivityLogs> GetAllLogs()
         {
             var activityLogs = activityLogRepository.GetAllLogs();
             
-            if (activityLogs == null)
-            {
-                var message = string.Format("activity Log not found");
-                throw new HttpResponseException(
-                    Request.CreateErrorResponse(HttpStatusCode.NotFound, message));
+            if (activityLogs == null){
```

Comment:
The formatting here is inconsistent with elsewhere in the code.  Elsewhere, the opening bracket appears on a separate line:

```C#
if (activityLogs == null)
{
    ...
```

Should we add a chore to add a code linter to the .NET build process?